deploy/operator.yaml

```diff
@@ -2738,7 +2738,7 @@ spec:
               name: "metrics"
               protocol: "TCP"
           # Replace this with the built image name
-          image: quay.io/kubevirt/hostpath-provisioner-operator:latest
+          image: registry:5000/hostpath-provisioner-operator:latest
```

Comment:
why change? local testing? going to introduce issue in #169?

Reply:
local testing, I didn't mean to check in the file.